src/env/log_fd.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2017-present, PingCAP, Inc. Licensed under Apache-2.0.
+
+#[cfg(any(target_os = "unix", target_os = "linux"))]
```

Comment:
I believe `target_os = "unix"` is a noop. The correct one should be `cfg(target_family = "unix")` or simply `cfg(unix)`.

Reply:
Also, we need a user feature flag to force enable std. Because our CI only run coverage check on Linux, so we must enable this flag to cover the newly added code.

I wish we can hide this flag from user docs though. The next best option is to just name it like "_use_std_fs" or something.

Reply:
Yes